keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -437,46 +437,38 @@ Operation metrics from [csiOperationsLatencyMetric](https://github.com/kubernete
 
 #### VolumeAttributesClass Deletion Protection
 
-While a VolumeAttributesClass is referenced by any PVC, we will prevent the object from being deleted by adding a finalizer([reference](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go)). 
```

Comment:
I remove the reference to the admission controller. It will make users confused if we don't use it. 
cc @msau42 updated. please review it again.